test/test_remote.py

```diff
@@ -175,6 +175,14 @@ def test_remote_list(self):
         remote = self.repo.create_remote(name, url)
         assert remote.name in [x.name for x in self.repo.remotes]
 
+    def test_ls_remotes(self):
+        assert 1 == len(self.repo.remotes)
+        remote = self.repo.remotes[0]
+
+        refs = remote.ls_remotes()
+
+        assert refs
```

Comment:
May it be possible to test the contents of `refs` a little bit? Like the expected dict keys are present.